modules/ROOT/pages/vpn-troubleshooting.adoc

```diff
@@ -0,0 +1,39 @@
+= Troubleshoot Anypoint VPN
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+These are some of the possible errors you may encounter when attempting to connect Anypoint VPN.
+
+=== Unable to Connect to the VPN
```

Comment:
Why jump from H1 to H3 (= to ===)?

Reply:
The headings just look weird being that big. What I really want is a DL, but AFAIK we don't have that ability.